guides/common/modules/proc_managing-and-monitoring-hosts-using-cockpit.adoc

```diff
@@ -22,8 +18,17 @@ For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{conte
 
+.Additional resources
+ifndef::satellite,orcharhino[]
+* For more information about {the-Cockpit}, see https://cockpit-project.org/documentation.html[Cockpit documentation].
+endif::[]
 ifdef::satellite[]
-For more information about getting started with Red Hat web console, see {RHELDocsBaseURL}8/html/managing_systems_using_the_rhel_8_web_console/index[_Managing systems using the RHEL{nbsp}8 web console_] or {RHELDocsBaseURL}7/html/managing_systems_using_the_rhel_7_web_console/getting-started-with-the-rhel-web-console_system-management-using-the-rhel-7-web-console#installing-the-web-console_getting-started-with-the-web-console[_Managing systems using the RHEL 7 web console_].
-
-For more information about using {LoraxCompose} through {the-Cockpit}, see {RHELDocsBaseURL}8/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-web-console-interface_composing-a-customized-rhel-system-image#accessing-composer-gui-in-the-rhel-8-web-console_creating-system-images-with-composer-web-console-interface[_Accessing Image Builder GUI in the RHEL{nbsp}8 web console_] or {RHELDocsBaseURL}7/html/image_builder_guide/chap-documentation-image_builder-test_chapter_4#sect-Documentation-Image_Builder-Chapter4[_Accessing Image Builder GUI in the RHEL{nbsp}7 web console_].
+* For more information about using {the-Cockpit}, see the following documents:
+** {RHELDocsBaseURL}9/html/managing_systems_using_the_rhel_9_web_console/index[_Managing systems using the RHEL{nbsp}9 web console_]
+** {RHELDocsBaseURL}8/html/managing_systems_using_the_rhel_8_web_console/index[_Managing systems using the RHEL{nbsp}8 web console_]
+** {RHELDocsBaseURL}7/html/managing_systems_using_the_rhel_7_web_console/getting-started-with-the-rhel-web-console_system-management-using-the-rhel-7-web-console#installing-the-web-console_getting-started-with-the-web-console[_Managing systems using the RHEL 7 web console_]
+* For more information about using {LoraxCompose} through {the-Cockpit}, see the following documents:
+** {RHELDocsBaseURL}9/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-web-console-interface_composing-a-customized-rhel-system-image#accessing-composer-gui-in-the-rhel-8-web-console_creating-system-images-with-composer-web-console-interface[_Accessing Image Builder GUI in the RHEL{nbsp}9 web console_]
+** {RHELDocsBaseURL}8/html/composing_a_customized_rhel_system_image/creating-system-images-with-composer-web-console-interface_composing-a-customized-rhel-system-image#accessing-composer-gui-in-the-rhel-8-web-console_creating-system-images-with-composer-web-console-interface[_Accessing Image Builder GUI in the RHEL{nbsp}8 web console_]
+** {RHELDocsBaseURL}7/html/image_builder_guide/chap-documentation-image_builder-test_chapter_4#sect-Documentation-Image_Builder-Chapter4[_Accessing Image Builder GUI in the RHEL{nbsp}7 web console_]
```

Comment:
nitpick:
When following the rhel7 link, one lands at `Accessing Image Builder GUI in the RHEL 7 web console`, however both rhel 8 and 9 take you to `Accessing the RHEL image builder dashboard in the RHEL web console`, should the link be name exactly as the section of the docs it targets?

Reply:
You are right and this is a good catch, thank you. I updated the RHEL 8 and 9 links to match the respective section headings.